home-manager/services/code-syncer/sync.sh

```
@@ -1,94 +1,336 @@
+
+# 3. Sync Extensions (with filtering)
+install_extensions "antigravity"
+install_extensions "cursor"
+install_extensions "windsurf"
+
+echo "Initial Sync Complete."
+
+# 4. Watcher (Mac only)
+if command -v fswatch >/dev/null; then
+  echo "Watching for changes in VS Code settings..."
+  fswatch -o "$VSCODE_USER_DIR/$SETTINGS_FILE" "$VSCODE_USER_DIR/$KEYBINDINGS_FILE" | while read num; do
```

Shellcheck (0.11.0)

[warning] num appears unused. Verify use (or export if used externally).

(SC2034)